tests/storage_test.go

```diff
@@ -46,7 +46,7 @@ var _ = Describe("Storage", func() {
 	})
 
 	getTargetLogs := func(tailLines int64) string {
-		pods, err := virtClient.CoreV1().Pods(metav1.NamespaceSystem).List(metav1.ListOptions{LabelSelector: "app in (iscsi-demo-target)"})
+		pods, err := virtClient.CoreV1().Pods("demo").List(metav1.ListOptions{LabelSelector: "app in (iscsi-demo-target)"})
```

Comment:
Instead of using a string literal, can we put this namespace in a const value in tests/utils in case we need to reference it elsewhere.